docs/en/install-upgrade/release-notes/release-notes-kibana.asciidoc

```diff
@@ -128,74 +159,47 @@ It will no longer be possible to create new scripted fields directly from the *D
 Migrate to runtime fields or ES|QL instead of creating new scripted fields. Existing scripted fields can still be edited or deleted.
 ====
 
-//Needs to be added to upgrade notes and detailed           
+
+
+**Elastic Observability solution**
+
+//Needs to be added to upgrade notes and detailed        
 [discrete]
-[[breaking-201254]]
-.Removed the legacy table in Discover.
+[[breaking-202278]]
+.Profiling now defaults to 19Hz sampling frequency.
 [%collapsible]
 ====
 *Details* +
-It's no longer possible to use the legacy documents table in Discover. To that effect, the `doc_table:legacy` and `truncate:maxHeight` deprecated advanced settings have been removed. For more information, refer to ({kibana-pull}201254[#201254]).
+For more information, refer to ({kibana-pull}202278[#202278]).
 ====
 
-//Needs to be added to upgrade notes and detailed           
+//Needs to be added to upgrade notes and detailed
 [discrete]
-[[breaking-200834]]
-.Now using Kibana feature privileges only to control access to reporting features.
+[[deprecation-203996]]
+.Disabled log stream and settings pages.
 [%collapsible]
 ====
 *Details* +
-The default system of granting users the privilege to generate reports has changed. Rather than assigning users the `reporting_user` role, administrators should create a custom role that grants report-creation privileges using Kibana application privileges. For more information, refer to ({kibana-pull}200834[#200834]).
+Logs Stream and the logs settings page in Observability are removed. Use the Discover application, which now offers a contextual experience for logs, to explore your logs. The logs stream panel in dashboards is removed, use Discover sessions instead.
+
+For more information, refer to ({kibana-pull}203996[#203996]).
```

Comment:
@mdbirnstiehl since the log settings page was removed, users that had updated their [settings for log sources](https://www.elastic.co/guide/en/enterprise-search/current/logging-view-query-logs.html#:~:text=Each%20log%20source%20integrates%20with,of%20a%20Kibana%20index%20pattern.) will need to migrate them to the new setting. This can be done through the upgrade assistant as [part of the upgrade](https://github.com/elastic/kibana/issues/186824) or [later through related callouts](https://github.com/elastic/kibana/pull/208242) or by manually updating the Kibana advanced setting. (https://github.com/elastic/kibana/pull/208242). Shall we add some information here as well?